### PR TITLE
Update the readme slightly

### PR DESCRIPTION
**Pull request by SomeoneToIgnore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +118,16 @@ Interrupt => sysboot ${devtype} ${devnum}:11 any ${scriptaddr} /boot/extlinux.co
 ```
 And that would boot our system on partition 17 (11 in base 16).
 
+Alternatively, use the following set of commands:
```
Comment (not-a-doctor-stromberg) [inline README.md:121]: I'd advocate for leaving this out, not sure what it adds.

File: README.md [hunk 2]
```diff
@@ -88,17 +95,19 @@ finalsetup.yaml
...
+Then, to install the rootfs using my laptop connected to the PineNote booted in Android, I do:
 ```
 $ adb push finalsetup.tar.gz /sdcard/Download
 $ adb shell
 $ su
 # mkdir /sdcard/target
 # mount /dev/block/mmcblk0p17 /sdcard/target
 # cd /sdcard/Download
-# tar xzf finalsetup.tar.gz -C /sdcard/target
+# busybox tar xzf finalsetup.tar.gz -C /sdcard/target
```
Comment (not-a-doctor-stromberg) [inline README.md:110]: is busybox definitely required for tar here? I don't recall needing it, but I'd trust you ;)

File: README.md [hunk 3]
```diff
@@ -88,17 +95,19 @@ finalsetup.yaml
 The work done by each recipe is saved as a `.tar.gz` file. So you would take the last archive, `finalsetup.tar.gz`, to extract on the PineNote. Currently, the archive's size is about 240MB and extracted into the partition would occupy almost 700MB.
 
 ## Install the rootfs on the PineNote
-Basically, you have to extract the `finalsetup.tar.gz` inside the prepared partition. You need to follow Martyn's and Dorian's guides to get to this point.
+Take the `*.tag.gz` file after the latest step and extract it to the prepared partition.
+You need to follow Martyn's and Dorian's guides to get to this point.
```
Comment (not-a-doctor-stromberg) [inline README.md:99]: Do you mind adding a link to Dorian's / Martyn's guide?

**Review by not-a-doctor-stromberg — COMMENTED** [review]
TY for improving the readme!

@m-weigand I added some comments that I thought were helpful, sorry if you'd rather I not do this :) just trying to help!

File: README.md (hunk 3 above)
Comment (SomeoneToIgnore) [reply README.md:99]: It is in the top of the README already, not sure it's worth repeating: 
image

File: README.md (hunk 1 above)
Comment (SomeoneToIgnore) [reply README.md:121]: This is actually from the latest release notes: https://github.com/m-weigand/pinenote-debian-recipes/releases/tag/v0.3-beta1

I've followed them and did not try the original command ever 🙂

**Comment by m-weigand** [comment]
Thanks for these changes. I think at one point we will need to restructure the readme...

**Pull request merged**